doc-content/enterprise-only/fuse/fuse-con.adoc

```diff
@@ -20,11 +20,21 @@ You can install {FUSE_LONG} on a separate instance of {EAP_LONG} and integrate i
 
 [IMPORTANT]
 ====
-For the version of {FUSE_LONG} that {PRODUCT} {ENTERPRISE_VERSION} supports, see 
+For the version of {FUSE_LONG} that {PRODUCT} {ENTERPRISE_VERSION} supports, see
 ifdef::PAM[]
 https://access.redhat.com/articles/3405381[{PRODUCT} 7 Supported Configurations].
 endif::[]
 ifdef::DM[]
 https://access.redhat.com/articles/3354301[{PRODUCT} 7 Supported Configurations].
 endif::[]
 ====
+
+[NOTE]
+====
+You can also install {FUSE_LONG} on SpringBoot. {PRODUCT} provides no special integration for this scenario. However, your application that runs on {FUSE_LONG} on SpringBoot can use the `kie-server-client` library to communicate with {PRODUCT} services that run on a {KIE_SERVER}. The following limitations apply, because of library version conflicts between {FUSE_LONG} and {PRODUCT}:
+
+* You cannot use `MarshallingFormat.JSON`, because of incompatible versions of Jackson.
```

Comment:
[style] Maybe avoid "You cannot use" and rephrase as per prerequisites statements?  Is it mismatched versions or incompatible version ranges? Is this a temporary version mismatch?
````
s/* You cannot use `MarshallingFormat.JSON`, because of incompatible versions of Jackson./* *Jackson version mismatch:* `MarshallingFormat.JSON` cannot be used.
````

Reply:
I do not think introducing the passive voice is better. (I did apply the other suggestions)